baby-bear/src/baby_bear.rs

```diff
@@ -396,6 +396,23 @@ const fn to_monty(x: u32) -> u32 {
     (((x as u64) << MONTY_BITS) % P as u64) as u32
 }
 
+/// Convert a constant u32 array into a constant Babybear array.
+/// Saves every element in Monty Form
+#[inline]
+#[must_use]
+pub(crate) const fn to_babybear_array<const N: usize>(input: [u32; N]) -> [BabyBear; N] {
+    let mut output = [BabyBear { value: 0 }; N];
+    let mut i = 0;
+    loop {
```

Comment:
A `for` loop would do the job and is more succinct

Reply:
I agree but I didn't think you could use `for` loops in constant functions.

Reply:
Right, I haven't seen that this was a constant function. Let me resolve the comment.